images/primers/docs/A2/index.md

```diff
@@ -0,0 +1,146 @@
+        "[fill='#FF00F5']": defineDOMSurface({
+          width: 1280,
+          height: 720,
+          useChromestage,
+        }),
+      },
+      angles: Object.fromEntries(await Promise.all([
+        ['grad_010_ovr', FileAttachment("../templates/grad_010_ovr.svg")],
+        ['grad_011_ovr', FileAttachment("../templates/grad_011_ovr.svg")],
+        ['grad_020_scr', FileAttachment("../templates/grad_020_scr.svg")],
+        ['grad_021_scr', FileAttachment("../templates/grad_021_scr.svg")],
+        ['grad_030_cha', FileAttachment("../templates/grad_030_cha.svg")],
+        ['grad_031_cha', FileAttachment("../templates/grad_031_cha.svg")],
+        ['grad_032_cha', FileAttachment("../templates/grad_032_cha.svg")],
+        ['grad_033_cu', FileAttachment("../templates/grad_033_CU.svg")],
+        ['grad_034_cu', FileAttachment("../templates/grad_034_CU.svg")],
+        ['grad_035_cu', FileAttachment("../templates/grad_035_CU.svg")],
+        ['grad_036_cu', FileAttachment("../templates/grad_036_CU.svg")],
+        ['grad_037_cu', FileAttachment("../templates/grad_037_CU.svg")],
+        ['grad_038_cha', FileAttachment("../templates/grad_038_cha.svg")],
+      ].map(async ([k, fa]) => [k, await fa.text()]))),
+    }),
+  },
+  events: [
+    {
+      "[fill='#00FF00']": {
+        keyframe: {
+          url: `${substrateURL}/bridge2;sessions=sp-01HR5B0SC9AT42Q3CC0S8C77S1`
+        },
+      },
+      "[fill='#FF00F5']": {
+        keyframe: {
+          url: `https://www.youtube.com/embed/0Qa5QvB40q4`,
+          youtube: {
+            currentTime: 1000,
+            playbackRate: 1,
+            playing: true,
+          },
+        },
+      },
+      'v1': {'angle': 'grad_010_ovr'},
+      'dur': 5000,
+      'notes': `Liv sits at her desk. She is typing up a class project as she watches her lecture for the day.`
+    },
+    {
+      'v1': {'angle': 'grad_021_scr'},
+      'dur': 4000,
+      'notes': `The lecture is a video. It’s shown full screen on her secondary monitor.`
+    },
+    {
+      'v1': {'angle': 'grad_020_scr'},
+      'dur': 3000,
+      'notes': `Focusing in on the lecture video, we see the subject matter is dense and requires external sources to be referenced.`
+    },
+    {
+      'v1': {'angle': 'grad_033_cu'},
+      'dur': 5000,
+      'notes': `Liv is typing in her document on her laptop screen. We see her trying her best to keep up with the lecture, feverishly taking notes and trying to absorb all the information.`
+    },
+    {
+      'v1': {'angle': 'grad_038_cha'},
+      'dur': 2500,
+      'notes': `A close-up of Liv’s face. She is visibly stressed from the effort she’s exerting.`,
+    },
+    {
+      'v1': {'angle': 'grad_030_cha'},
+      'dur': 7000,
+      'notes': `Liv continues to type, but then she realizes that getting Bridge's help would be a good idea.`,
+      'dialog': {
+        'Liv': `Bridge, take note of specific people and events mentioned so I can follow up on them later`
```

Comment:
@yoshikiohshima here's where Liv talks to Bridge. Did I understand your idea that Liv should pause the video before talking to Bridge?